src/renderChat.tsx

```diff
@@ -24,7 +24,17 @@ export const renderChat: (
 ): void => {
   ReactDOM.render(
     <ThemeProvider theme={createTheme(theme)}>
-      <TockContext>
+      <TockContext
+        {...(storageAvailable('localStorage') && {
+          localStorageHistory: {
+            enable:
+              options.localStorage ||
+              options.localStorageHistory?.enable === true,
+            maxNumberMessages: options.localStorageHistory?.maxNumberMessages,
+            storagePrefix: options.localStorageHistory?.storagePrefix,
+          },
+        })}
+      >
```

Comment:
On further inspection, we should really try to avoid this duplication. We could simply shove the logic in a method, but it would still negatively impact React users, who will have to call the right method with the right parameters twice in their render tree. Furthermore, it seems likely that some other features may need the localStorage prefix at some point, so we should consider a way to make it accessible globally.

I believe the right way forward is to add a `config` field to the `TockState`, which would at first only contain this new `storagePrefix` (inside a `localStorage` namespace for future-proofing). Said config would be passed as a property to the `TockContext`, replacing the `localStorageHistory` parameter added in this PR. We could consider hoisting the whole `localStorageHistory` object to that config, but it may best be done as a second PR, as it would also be a breaking change to the `useTock` interface.